styles/styles.css

```diff
@@ -27,4 +27,44 @@ body {
 /*** TODO: side bar ***/
 
 /*** TODO: content ***/
+
+.standard-page-content-wrapper {}
+.standard-page-content-section {}
```

Comment:
Remove `.standard-page-content-wrapper` and `.standard-page-content-section` since there's no styling applied for those.